src/components/types.ts

```diff
@@ -152,6 +152,15 @@ export interface VideoFormat extends MediaFormat {
   poster?: string
 }
 
+export interface MultipartFormat extends DataFormat {
+  /** An array of file names. */
+  files: string[]
```

Comment:
I'm wondering if this field should be an array of objects that stores the name and url of each file.

Reply:
We could update files to be an array of objects that stores the name and url of each file. The url should be optional.